packages/workspace/src/browser/workspace-uri-contribution.ts

```diff
@@ -32,7 +32,14 @@ export class WorkspaceUriLabelProviderContribution extends DefaultUriLabelProvid
 
     @postConstruct()
     protected async init(): Promise<void> {
-        // no-op, backward compatibility
+        const outer = this;
+
+        setInterval(() => {
+            outer.x++;
+            for (const uri of outer.testUris.values()) {
+                outer.onElementUpdatedEmitter.fire(uri);
```

Comment:
i would imagine here:
```ts
fire({
    affects: element => outer.testUris.has(getUri(element))
});
```

Reply:
I've changed it.  (though see my comment below saying that we could just test for 'test' here and do away with the map)